lib/msprime.c

```diff
@@ -3787,6 +3795,13 @@ msp_merge_two_ancestors(msp_t *self, population_id_t population_id, label_id_t l
         goto out;
     }
 
+    uint32_t min_overlap;
```

Comment:
local style here says that we keep variable declarations in the first block of the function. So, this would join the other delcarations above

```
uint32_t min_overlap = self->stop_at_local_mrca? 2: 0;
```